go/registry/api/api.go

```diff
@@ -303,12 +303,18 @@ type NodeUnfrozenEvent struct {
 	NodeID signature.PublicKey `json:"node_id"`
 }
 
+// NodeListEpochEvent is the per-epoch node list event.
```

Comment:
I don't think we should expose this?

Reply:
agreed, I added an `InternalEvent` type for registry that includes the internal events that registry backend needs to process but aren't exposed (currently only such event type is the NodeListEpochEvent).